src/Microsoft.Dnx.Runtime/PathSearchBasedProjectResolver.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Microsoft.Dnx.Runtime
+{
+    public class PathSearchBasedProjectResolver : IProjectResolver
```

Comment:
Because `ApplicationHostContext` need to fall back to it in DTH. VS could open a project which is not restored. Under that situation lock file doesn't exist, project resolving will relies on path search.